packages/lib/actionClient/permissions.ts

```diff
@@ -2,69 +2,270 @@ import { ZProductUpdateInput } from "@formbricks/types/product";
 
 export const Permissions = {
   owner: {
+    environment: {
+      read: true,
```

Comment:
Not sure if this permission makes sense, since every user needs to be able to read the environment, otherwise using formbricks is impossible 😉

Reply:
I added this to make the `checkAuthorization` check consistent. This is more of a redundant check, but organizationId is also being checked based on the environment when this permission is being used.